library/build.gradle

```diff
@@ -1,6 +1,6 @@
 apply plugin: 'com.android.library'
-apply plugin: 'maven'
-apply plugin: 'signing'
+// for Android libraries
+apply plugin: 'com.github.dcendents.android-maven'
```

Comment:
@BenHenning i have used https://github.com/oppia/androidsvg since this is a java project i am getting alot of build errors if i use kotlitex.

Reply:
Please see screenshot of error while building after changing build.gradle to match kotlitex library.
<img width="1392" alt="Screenshot 2022-09-08 at 16 57 59" src="https://user-images.githubusercontent.com/20886444/189141563-040a0016-b5b8-4de9-99e2-63517f154b9f.png">

Reply:
@KevinGitonga can you push the changes or provide a Gist to show what you've specifically tried that led to that error?

Reply:
@BenHenning i have updated for some reason i had to remove some tags on the initial library to get it to build successfully also updated Android api level declared on the manifest files PTAL.

Reply:
After this commit not able to build successfully on Jitpack below are the logs https://jitpack.io/com/github/KevinGitonga/Android-MarqueeView/48c8e087f1/build.log

Reply:
Thanks. Does the build actually work locally for you? Per Jitpack, it's trying to run: ``./gradlew clean -Pgroup=com.github.KevinGitonga -Pversion=48c8e087f1 -xtest -xlint assemble publishToMavenLocal``.

I think the part that's actually relevant to try is ``./gradlew assemble publishToMavenLocal``. I'm assuming this will fail if you try it locally, so you can probably use it to debug whatever's wrong with the configuration without needing to push and way for Jitpack. https://docs.gradle.org/current/userguide/publishing_maven.html may also help with trying to figure out what's wrong here.

Reply:
Thanks for this pointer, trying to make it work locally before pushing the changes.

Reply:
Given your update on the Oppia Android PR, is this working now on Jitpack?

Reply:
Yeah @BenHenning this is working as excepted.